yarn-project/aztec-rpc/src/aztec_rpc_server/aztec_rpc_server.ts

```diff
@@ -112,17 +122,15 @@ export class AztecRPCServer implements AztecRPCClient {
     }
 
     const flatArgs = encodeArguments(constructorAbi, args);
-
-    const fromAddress = from.equals(AztecAddress.ZERO) ? (await this.keyStore.getAccounts())[0] : from;
+    const addressSalt = contractAddressSalt || Fr.random();
```

Comment:
hmm we don't want to just do contractAddressSalt = contractAddressSalt || Fr.random()?

Reply:
What about just putting the default in the args then? contractAddressSalt=Fr.random() instead of being just a question mark param

Reply:
I don't either, but I'll choose it over an artificial name

Reply:
I literally just changed it to a default random value before reading your comment haha.